.pipelines/.vsts-vhd-builder-pr-windows.yaml

```diff
@@ -59,3 +59,5 @@ stages:
       build23H2gen2: True
       overrideBranch: master
       useOverrides: False
+      build2025: False
+      build2025gen2: False
```

Comment:
One of these should be set to true so we don't break 2025 VHDs. Prob the gen2 one as that's the default.